packages/vector_graphics/lib/src/render_vector_graphic.dart

```diff
@@ -439,7 +439,7 @@ class RenderPictureVectorGraphic extends RenderBox {
       context.canvas.translate(offset.dx, offset.dy);
     }
     if (_opacityValue != 1.0 || colorFilter != null) {
-      context.canvas.saveLayer(offset & size, colorPaint);
+      context.canvas.saveLayer(Offset.zero & size, colorPaint);
```

Comment:
What about just passing null? I think that's usually what we want anywya, since that means Skia will just use the "current clip" bounds, and we won't accidentally introduce a different clip here unintentionally.

Reply:
That expands to the full screen in simple examples

Reply:
Ahh ok, makes sense.